### ISSUE TITLE
Fermented foods

### ISSUE THREAD

**Issue by scbrinkley** [FoodOntology/foodon#320]
Hello FoodOn, I'm finding some missing fermented foods. When I query fermented, I would expect to find cheese, chocolate, coffee, all products of fermentation, with a fermented designation. Whether the foods are a multi-component food product or food product by process, *food (fermented)* does not contain any of the above food. Though I do see some annotations under certain fermentation products, take romano cheese (pecorino-style) (http://purl.obolibrary.org/obo/FOODON_03305231) for example, there is an annotation:

http://www.w3.org/2000/01/rdf-schema#comment:
SIREN DB annotation: * solid FOODON:03430151 * curd FOODON:03420245 * lactic acid-other agent fermentation process FOODON:03460107 

Does this indicate a mapping of all products of fermentation is underway?

**Comment by maweber-bia** [comment]
The process of fermentation encompasses the biochemical activity of organisms throughout their life cycle, from growth to death. https://pmc.ncbi.nlm.nih.gov/articles/PMC10991178/

**Comment by ddooley** [comment]
Discussed on Nov 14th call.  Action items:
* "**food fermentation**" needs its definition cleaned up: "A fermentation process in which either carbohydrates, proteins or fats are modified through microbial, enzymatic and/or other biological process."

  Proposed revision: "A [microbial/enzymatic modification process](https://ontobee.org/ontology/FOODON?iri=http://purl.obolibrary.org/obo/FOODON_03460119) in which either carbohydrates, proteins or fats are modified through a microbial metabolic process."

* "**[microbial/enzymatic modification process](https://ontobee.org/ontology/FOODON?iri=http://purl.obolibrary.org/obo/FOODON_03460119)**" needs a label tweak: " a microbial or enzymatic modification process".

It appears not all "cheese" is fermented, e.g. "farmers cheese", Paneer - acid coagulated milk cheese.
Determine if curds is a pre-fermented stage?

Looks like we need two top level cheese classes:

* Fermented cheese: (necessarily aged?)
* Non-fermented cheese: (is a subclass of unripened cheese?)

@maweber-bia Work will be done on a new Fermentation tab of [FoodOn Robot Tables](https://docs.google.com/spreadsheets/d/1VJtz4m67tdUNDqRe3m1Okdxll64nTR46GSvCOmb0APE)

Papers:

[An overview of fermentation in the food industry - looking back from a new perspective](https://pmc.ncbi.nlm.nih.gov/articles/PMC10991178/)
[The periodic table of fermented foods: limitations and opportunities](https://doi.org/10.1007/s00253-022-11909-y)  

Task can encompass ensuring a good organization for all fermented foods in FoodOn.

**Comment by scbrinkley** [comment]
From Sandor Katz, the Art of Fermentation:

> [...] fermentation is the transformation of food by various bacteria, fungi, and the enzymes they produce.

Although I would argue it is the bacteria **and/ or** fungi (and/or other micro-organisms) and the enzymes they produce. 

Formally, biologists describe fermentation as anaerobic redox metabolism (or the production of energy in absence of oxygen), but I don't think that's what we are going for here.

To comment on the part about "carbohydrates, proteins or fats", those aren't the only compound classes transformed in fermentation. Think: flavor and aroma compounds (McFeeters, 2004).

Katz, S.E., 2012. The art of fermentation: an in-depth exploration of essential concepts and processes from around the world. Chelsea green publishing.

Mcfeeters RF. Fermentation Microorganisms and Flavor Changes in Fermented Foods. Journal of Food Science (2004) 69: doi: [10.1111/j.1365-2621.2004.tb17876.x](https://doi.org/10.1111/j.1365-2621.2004.tb17876.x)

**Comment by maweber-bia** [comment]
> and the enzymes they produce.

yes but one should notice that fermentation implies that the microorganisms are alive ; enzymatic process only in not a "fermentation" process but a biochemical reaction occuring in the food matrix

and yes, biologically,  fermentation is an anaerobic redox metabolism which can follows different pathways producing various metabolites (alcohols,  lactic acid,  acetic acid, etc...)

**Comment by maweber-bia** [comment]
> Looks like we need two top level cheese classes:
> 
>     * Fermented cheese: (necessarily aged?)
> 
>     * Non-fermented cheese: (is a subclass of unripened cheese?)

We need to take into account 2 main stages in cheese-making
- acidification using lactic ferments (either wild or added)
- ripening using a variety of microorganisms (bacteria, yeasts, molds), depending on the type of ripening process / cheese family
so you can have fermented (fresh=unripened) cheese or fermented ripened cheese

so fermented cheese is not necessarily "aged"

and non-fermented cheese could be a subclass of unripenened cheese --> non-fermented cheese would be any cheese obtained by adding chemicals (no action by bacterial/microbial agent in this case)

**Comment by oldskeptic** [comment]
Final Suggestion: "Fermentation is a transformation of matter by various bacteria, fungi, and the enzymes they produce through a microbial metabolic processes." @ddooley

**Comment by maweber-bia** [comment]
> Final Suggestion: "Fermentation is a transformation of matter by various bacteria, fungi, and the enzymes they produce through a microbial metabolic processes." @ddooley

sorry,I would like to make another slight change:

Fermentation is a transformation of organic substances by various **living microorganisms** and the enzymes they produce through a  metabolic processes."

**Comment by ddooley** [comment]
Discussed Dec 12: FoodOn will have a general fermentation definition as Magalie suggests, and then under, will be "food fermentation", but with siblings potentially in future (or now) of "alcoholic fermentation", "lactic fermentation" etc. which are not planned processes necessarily.  The GO [fermentation](http://purl.obolibrary.org/obo/GO_0006113) term isn't sufficient because it specifically mentions anaerobic whereas aerobic fermentation needs to be covered too.  

Recommend that GO change "fermentation" label to "anaerobic fermentation".

CHMO "[fermentation](http://purl.obolibrary.org/obo/CHMO_0001624) mentions bioreactor specifically, and a planned process, so we recommend CHML change the label to "bioreactor fermentation".

We'll see who might take on fermentation at the "process" parent level, rather than planned process.

@ddooley will follow up with GO and CHMO about this.

That makes "food fermentation" a planned process.

**Comment by nbokulich** [comment]
As a general definition of food fermentation I recommend using the [definition from ISAPP](https://www.nature.com/articles/s41575-020-00390-5 ) that these are "foods made through desired microbial growth and enzymatic conversions of food components" . This is very similar to what is proposed above.

**Comment by maweber-bia** [comment]
yes, but no ! I think it is quite important to keep the notion of **living** microorganism in the definition, as stated in the paper from ISAPP:

"**The definition requires the activity of microorganisms.** Although endogenous or exogenous enzymes from plants, animals or other sources might be present, the activities of those enzymes alone are insufficient for a food to be regarded as fermented. This definition is sufficiently broad to include not only the fermentations noted earlier but also to distinguish fermentation from its microbiological converse, namely food spoilage. Whereas both processes occur via microbial growth and enzymatic activity on food constituents, spoilage is clearly unintentional and fermentation is deliberate and controlled to generate the desirable attributes."


Here is also pointed out the **intentionality** of the food fermentation process, **in contrast to spoilage.**

However, in the paper it is stated that "The definition includes foods and beverages that are produced by fermentation but might not have living microorganisms **at the time of consumption.**"

**Comment by nbokulich** [comment]
> it is quite important to keep the notion of living microorganism in the definition

I agree! This is also implied by the ISAPP definition ("foods made through... microbial _growth_"). Growth implies that these microorganisms must be living during the fermentation process in order to ferment the food substrate. 

However, a definition of fermented food should not require live microorganisms in the finished product/at the time of consumption, because otherwise many many foods disqualify even though they are indeed fermented. Many fermented foods indeed lack viable microorganisms at the time of consumption, either because these foods are cooked before consumption (breads, many fermented meats and vegetables) or because microorganisms are lost/removed during downstream processing (e.g., wines, beers, pasteurized fermented foods). This does not make them unfermented. I suppose that's where subcategories in FoodON would be needed to differentiate subclasses of fermented foods, as well as key characteristics such as viability.

**Comment by maweber-bia** [comment]
Yes, we should distinguish between "food fermentation" which is a process and "fermented foods" which are the outcomes of the food fermentation process

**Comment by ddooley** [comment]
I've tweaked this slightly to fit Aristotelean form, influenced by GO's hierarchy: 

label: fermentation; defn: "**A metabolic process which transforms organic substances by various living microorganisms and the enzymes they produce**."  

But is there something we can say about what molecules the transformation leads to?  GO focuses on the creation of constituent compounds?

BTW this older "Fermentation (Industrial)" paper https://www.sciencedirect.com/sdfe/pdf/download/eid/3-s2.0-B0122270703005705/first-page-pdf introduces a nice set of distinctions - in context of solid-state or submerged cultures, both eating "substrate", and then talks about aerobic and anaerobic fermentation.

**Comment by maweber-bia** [comment]
> But is there something we can say about what molecules the transformation leads to? GO focuses on the creation of constituent compounds?

I think it is almost impossible to list all the possible outcomes! As stated in the "fermentation (industrial) basic considerations" paper you mentionned, fermentation processes utilize microorganisms to convert solid or liquid substrates into various products. The substrates used vary widely, any material that supports microbial growth being a possible substrate. Similarly, fermentation derived products show tremendous variety (see introduction section).

So, what we can say is that a variety of molecules and compounds are produced by cascade reactions from the breakdown of carbohydrates, proteins and fats, or other organic compounds.

And a lot of fun in 2025!

**Comment by maweber-bia** [comment]
Additional reference: Chisti, Y. (2010). Fermentation Technology. In Industrial Biotechnology (eds W. Soetaert and E.J. Vandamme). https://doi.org/10.1002/9783527630233.ch3

![image](https://github.com/user-attachments/assets/24c85b92-e6ce-4c73-b2a8-7f7a629083a1)
